src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,184 @@
+    private static final int MAX_TASKS_NUMBER = 100;
+    private static final Scanner INPUT_COMMAND = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        Task[] tasksList = new Task[MAX_TASKS_NUMBER];
+        int taskCount = 0;
+        showWelcomeMessage();
+        executeProgram(tasksList, taskCount);
+        exitProgram();
+    }
+
+    public static void executeProgram(Task[] tasksList, int taskCount) {
+        String inputLine = INPUT_COMMAND.nextLine();
+        ;
+        while (!inputLine.equals(COMMAND_EXIT_WORD)) {
+            try {
+                taskCount = executeCommand(tasksList, taskCount, inputLine.trim());
+                inputLine = INPUT_COMMAND.nextLine();
+            } catch (IllegalCommand e) {
+                System.out.println("  OOPS! I'm sorry, but I don't know what that means :(");
+                System.out.println("  I can only execute these commands: ");
+                for (int i = 1; i <= COMMAND_WORDS_LIST.length; i++) {
+                    System.out.println("  " + i + ". " + COMMAND_WORDS_LIST[i - 1]);
+                }
+                inputLine = INPUT_COMMAND.nextLine();
+            } catch (EmptyToDoDescription e) {
+                System.out.println("  OOPS! The description of a todo cannot be empty!");
+                System.out.println("  Please input again with the correct format!");
+                inputLine = INPUT_COMMAND.nextLine();
+            } catch (IllegalDeadlineInput e) {
+                System.out.println("  OOPS! Please input again using this format:!");
+                System.out.println("  'deadline <description> /by <due date>'");
+                inputLine = INPUT_COMMAND.nextLine();
+            } catch (IllegalEventInput e) {
+                System.out.println("  OOPS! Please input again using this format:!");
+                System.out.println("  'event <description> /at <event date>'");
+                inputLine = INPUT_COMMAND.nextLine();
+            } catch (EmptyTaskNumber e) {
+                System.out.println("  OOPS! Please specify the completed task number!");
+                System.out.println("  Example: 'done 1'");
+                inputLine = INPUT_COMMAND.nextLine();
+            } catch (NumberFormatException e) {
+                System.out.println("  OOPS! Please put only integer as the task number!");
+                System.out.println("  Example: 'done 1'");
+                inputLine = INPUT_COMMAND.nextLine();
+            } catch (NullPointerException e) {
+                System.out.println("  OOPS! You must have typed the wrong number. Please type in again correctly!");
+                inputLine = INPUT_COMMAND.nextLine();
+            }
+        }
+    }
+
+    public static int executeCommand(Task[] tasksList, int taskCount, String inputLine) throws IllegalCommand, EmptyToDoDescription, IllegalDeadlineInput, IllegalEventInput, EmptyTaskNumber {
```

Comment:
A line length should not be more than 120 characters because it will be harder for other programmers to look at the code. You should wrap a long line to the next line for better readability.